ssh/DOCS.md

```diff
@@ -255,8 +255,11 @@ Setting the following parameters as is shown will allow you to connect to
 your Home Assistant instance using VSCode Remote - SSH:
 
 ```yaml
-allow_remote_port_forwarding: true
-allow_tcp_forwarding: true
+ssh:
+  allow_remote_port_forwarding: true
+  allow_tcp_forwarding: true
+packages:
+  - procps
```

Comment:
Honestly, if this is the way to make newer version work, we should adjust the add-on. Not the documentation.

Reply:
I see. TBH, that wasn't my initial instinct as this addon is "Advanced SSH" and not "Visual Studio Code server in disguise" 😁. So it made sense to me to do the extra step in the configuration.
That said, if `procps` is already installed in the container, I guess it's even easier.

I edited the PR. I got the version to install by `apk info procps`, which actually talks about `procps-ng`, but it seems to work anyways. I'm not that familiar with Alpine, so feel free to correct/ask for corrections as you see fit.
And thanks for everything 🙂

Reply:
Nah this is fine 👍